### PR TITLE
bazel: Simplify select alias configuration

### PR DESCRIPTION
**Pull request by keith**
When this was added originally this skylib helper didn't exist.

Signed-off-by: Keith Smiley <keithbsmiley@gmail.com>

### CODE REVIEW

File: bazel/BUILD [hunk 1]
```diff
@@ -308,17 +308,17 @@ selects.config_setting_group(
 )
 
 config_setting(
-    name = "disable_hot_restart",
+    name = "disable_hot_restart_setting",
     values = {"define": "hot_restart=disabled"},
+    visibility = ["//visibility:private"],
```
Comment (mattklein123) [inline bazel/BUILD:313]: While you are in here can you potentially do this also for the http3 setting to make sure it's always accessed through the aggregate one I added?

**Review by mattklein123 — CHANGES_REQUESTED** [review]
Awesome. Check CI?

/wait

**Review by mattklein123 — APPROVED** [review]
Awesome. Did you want to make any more changes to this?

**Comment by keith** [comment]
nope I think we're good here, thanks!

**Pull request merged**